### PR TITLE
Add examples for supporting CSS Modules

### PR DESCRIPTION
**Pull request by ostowe**

### CODE REVIEW

File: localClasses.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+const path = require('path');
+const fs = require('fs-extra');
+
+module.exports = function localClasses(cssFileName, json) {
+  const cssFileInfo = path.parse(cssFileName);
+  const jsonFileName  = path.join('static/css/classNames', `${cssFileInfo.name}.json`);
```
Comment (meddy) [inline localClasses.js]: Do you want `__dirname` here?

File: static/prototype/huron-assets/index.js [hunk 2]
```diff
@@ -0,0 +1,86 @@
+/*eslint-disable*/
```
Comment (meddy) [inline static/prototype/huron-assets/index.js]: If you are specifying node 6, then why do you have to write this in ES5 style?

Comment (meddy) [reply static/prototype/huron-assets/index.js]: You are referencing window, so I guess this is a browser script.

File: static/prototype/huron-assets/index.js [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+  module.hot.accept(
+    assets.id,
+    () => {
+      var newAssets = require.context(
+        '../partials',
+        true,
+        /\.html|\.json|\.hbs$/
+      );
+      var newModules = newAssets.keys()
+        .map((key) => {
```
Comment (meddy) [inline static/prototype/huron-assets/index.js]: Wait... arrow function. That's ES6.

File: static/prototype/huron-assets/index.js [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+  }
+};
+
+function updateStore(newStore) {
+  insert.store = newStore;
+}
+
+
+
+/*eslint-enable*/
```
Comment (meddy) [inline static/prototype/huron-assets/index.js]: Maybe you should eslint this.

File: static/prototype/huron-assets/index.js (hunk 3 above)
Comment (ostowe) [reply static/prototype/huron-assets/index.js]: haha this is a built file

**Comment by ostowe** [comment]
Pretty sure you'd have about 1.5m pointers for me on the huron codebase. Someday perhaps.

File: static/prototype/huron-assets/index.js (hunk 3 above)
Comment (ostowe) [reply static/prototype/huron-assets/index.js]: Oh wait not it's not. Yeah, I'm not actually sure how to tackle this issue. It's JS that needs to be written programmatically, so I don't know how to lint or transpile it. It's like meta-js.

**Pull request merged**